ethergo/submitter/suite_test.go

```diff
@@ -91,8 +92,16 @@ func (s *SubmitterSuite) SetupSuite() {
 	go func() {
 		defer wg.Done()
 		var err error
-		localmetrics.SetupTestJaeger(s.GetSuiteContext(), s.T())
-		s.metrics, err = metrics.NewByType(s.GetSuiteContext(), buildInfo, metrics.Jaeger)
+		// don't use metrics on ci for integration tests
+		isCI := core.GetEnvBool("CI", false)
+		useMetrics := !isCI
+		metricsHandler := metrics.Null
+
+		if useMetrics {
```

Comment:
does this actually work?

Reply:
We observed it working when running `watch -n 1 'docker ps'` manually in upterm